doc/developers/performance.rst

```diff
@@ -388,8 +388,31 @@ Checkout the official joblib documentation:
 
 .. _warm-restarts:
```

Comment:
should we change all the `_warm-restarts` to `_warm-starts`? Better to have one fixed terminology.